numba_dpex/examples/debug/side-by-side.py

```diff
@@ -5,15 +5,16 @@
 import argparse
 
 import dpctl
+import dpnp
 import numba
 import numpy as np
 
 import numba_dpex as ndpx
 
 
 def common_loop_body(param_a, param_b):
-    param_c = param_a + 10  # Set breakpoint here
-    param_d = param_b * 0.5
+    param_c = param_a + np.float32(10)  # Set breakpoint here
```

Comment:
```suggestion
    param_c = param_a + param_a.dtype.type(10)  # Set breakpoint here
```

Same for other places. We want to avoid using library directly, since it may be used both ad numba(numpy) and numba_dpex(dpnp)